applications/FluidDynamicsApplication/python_scripts/navier_stokes_compressible_explicit_solver.py

```diff
@@ -50,6 +50,7 @@ def GetDefaultParameters(cls):
             },
             "echo_level": 1,
             "time_order": 2,
+            "runge_kutta_scheme" : "RK4",
```

Comment:
I'd call this `time_scheme` as we do in the other solvers.

Reply:
Agreed